events/teamJoin.js

```diff
@@ -1,14 +1,15 @@
 function teamJoin(context, payload) {
   const playerState = context.getPlayerState();
-  const playerId = playerState.id;
+  const playerId = context.id;
```

Comment:
Small nitpick: `context.id()`, I changed the `id` into a function at some stage (my bad!), forgot when :(

Reply:
omfg sorry that was some casual slarv